pkg/agent/baker.go

```diff
@@ -111,6 +111,42 @@ func (t *TemplateGenerator) getLinuxNodeCSECommand(config *datamodel.NodeBootstr
 	return strings.Replace(str, "\n", " ", -1)
 }
 
+// getWindowsNodeCSECommand returns Windows node custom script extension execution command
+func (t *TemplateGenerator) getWindowsNodeCSECommand(config *datamodel.NodeBootstrappingConfiguration) string {
+	var userAssignedIdentityClientIDParams string
+	isUserAssignedIdentity := config.ContainerService.Properties.OrchestratorProfile.KubernetesConfig.UserAssignedIDEnabled()
```

Comment:
Must this `config.ContainerService.Properties.OrchestratorProfile.KubernetesConfig` not be nil?

Reply:
Will add a nil check.